pkg/rtc/participant.go

```diff
@@ -1782,3 +1789,14 @@ func (p *ParticipantImpl) postRtcp(pkts []rtcp.Packet) {
 		p.params.Logger.Warnw("rtcp channel full", nil)
 	}
 }
+
+func (p *ParticipantImpl) setDowntracksConnected() {
+	p.lock.RLock()
+	defer p.lock.RUnlock()
+
+	for _, t := range p.subscribedTracks {
+		if dt := t.DownTrack(); dt != nil && dt.Kind() == webrtc.RTPCodecTypeVideo {
```

Comment:
Maybe don't restrict this by video type. `SetConnected` may not be useful for `audio`, but from an API and also internally in down track, connected state being different for audio/video would be confusing.